corehq/pillows/reportcase.py

```diff
@@ -40,49 +36,3 @@ def get_case_to_report_es_processor():
         doc_filter_fn=report_case_filter,
         change_filter_fn=is_couch_change_for_sql_domain
     )
-
-
-def get_report_case_to_elasticsearch_pillow(pillow_id='ReportCaseToElasticsearchPillow',
-                                            num_processes=1, process_num=0, **kwargs):
-    # todo; To remove after full rollout of https://github.com/dimagi/commcare-hq/pull/21329/
```

Comment:
This comment appears 12 times in the codebase.  Getting rid of the remaining 10 would be another great tech debt effort, though it'd require a bit more due diligence to ensure it won't affect any third parties.

Reply:
One thing that is tricky about that is that we use those single processor pillows in our reindexers (you can see a discussion of a recent attempt to get rid of a different one here)